CHANGELOG.rst

```diff
@@ -71,6 +71,7 @@ Protocol is now at version **1.16**. See `API changelog`_.
 - Removed Structlog binding and bottlenecks (fixes #603)
 - Fixed Swagger output with subpath and regex in pyramid routes (fixes #1180)
 - Fixed Postgresql errors when specifying empty values in querystring numeric filters. (fixes #1194)
+- Catch UnicodeDecodeError in case of request wrong encoding (fixes #1195)
```

Comment:
→ Return 400 bad request instead of crashing when querystring contains bad characters ?